### PR TITLE
[4.0][a11y] Extra column for icon featured in com_contact

### PR DESCRIPTION
**Pull request by hardik-codes**
### Summary of Changes
An extra column has been added for the icon featured in com_contact

### Testing Instructions
Go to Components -> Contacts
In the dropdown menu of Contacts, click on Contacts

### Expected result

com_contact 1

### Actual result
com_contact 2

### Documentation Changes Required
New Screen

### CODE REVIEW

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on 19d992c92e34254167102d6d477eb0db6cf744bf<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23885">issues.joomla.org/tracker/joomla-cms/23885</a>.</sub>

File: administrator/components/com_contact/tmpl/contacts/default.php [hunk 1]
```diff
@@ -126,10 +129,15 @@
...
 								<td class="text-center">
 									<div class="btn-group">
-										<?php echo HTMLHelper::_('jgrid.published', $item->published, $i, 'contacts.', $canChange, 'cb', $item->publish_up, $item->publish_down); ?>
-										<?php echo HTMLHelper::_('contactadministrator.featured', $item->featured, $i, $canChange); ?>
-									</div>
+									<?php echo HTMLHelper::_('contactadministrator.featured', $item->featured, $i, $canChange); ?>
+								   	</div>
+								</td>	
+								<td class="text-center">
+								    <div class="btn-group">
```
Comment (Quy) [inline administrator/components/com_contact/tmpl/contacts/default.php]:
```suggestion
									<div class="btn-group">
```

File: administrator/components/com_contact/tmpl/contacts/default.php [hunk 2]
```diff
@@ -126,10 +129,15 @@
...
+								    <?php echo HTMLHelper::_('jgrid.published', $item->published, $i, 'contacts.', $canChange, 'cb', $item->publish_up, $item->publish_down); ?>
```
Comment (Quy) [inline administrator/components/com_contact/tmpl/contacts/default.php]:
```suggestion
										<?php echo HTMLHelper::_('jgrid.published', $item->published, $i, 'contacts.', $canChange, 'cb', $item->publish_up, $item->publish_down); ?>
```

**Comment by C-Lodder** [comment]
I have tested this item :white_check_mark: successfully on 19d992c92e34254167102d6d477eb0db6cf744bf<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23885">issues.joomla.org/tracker/joomla-cms/23885</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23885">issues.joomla.org/tracker/joomla-cms/23885</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**